pkg/kubectl/cmd/create_job.go

```diff
@@ -138,9 +140,11 @@ func (c *CreateJobOptions) createJob(cronJob *batchv1beta1.CronJob) error {
 		Spec: cronJob.Spec.JobTemplate.Spec,
 	}
 
-	job, err := c.Client.Jobs(c.Namespace).Create(jobToCreate)
-	if err != nil {
-		return fmt.Errorf("failed to create job: %v", err)
+	if !c.DryRun {
+		_, err := c.Client.Jobs(c.Namespace).Create(job)
```

Comment:
Seems like we should output the object that was created.

Reply:
This is still not addressed, we want to print the actual object that got created.